cucoslib/workers/gh_metadata_result_collector.py

```diff
@@ -0,0 +1,32 @@
+from selinon import StoragePool
+from cucoslib.base import BaseTask
+from cucoslib.utils import get_analysis_by_id
+
+
+class GitHubManifestMetadataResultCollector(BaseTask):
+    """
+    Collect all results that were computed, upload them to S3 and store version reference to results in WorkerResult
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def run(self, arguments):
+        self._strict_assert(arguments.get('ecosystem'))
+        self._strict_assert(arguments.get('repo_name'))
```

Comment:
Why there is `repo_name`? Based on the job handler it is basically a duplicity of the repo URL.